plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c

```diff
@@ -1206,7 +1207,7 @@ fu_logitech_bulkcontroller_device_transition_to_device_mode(FuLogitechBulkcontro
 {
 	return fu_device_retry(FU_DEVICE(self),
 			       fu_logitech_bulkcontroller_device_transition_to_device_mode_cb,
-			       MAX_RETRIES,
+			       MAX_SETUP_RETRIES,
```

Comment:
how long (in time, worst case) does this take? I'm worried it's going to block fwupd startup for minutes at startup if the hardware is wedged.

Reply:
Slowest hardware succeeded after ~10 retries. Took < 25 sec.